lib/hutch/acknowledgements/nack_on_all_failures.rb

```diff
@@ -0,0 +1,18 @@
+require 'hutch/logging'
+
+module Hutch
+  module Acknowledgements
+    class NackOnAllFailures
+      include Logging
+
+      def handle(delivery_info, properties, broker, ex)
```

Comment:
Since the interface here is a single function, it makes more sense to use `call` so that functions (blocks, procs, etc) can be used as well.